WordPress/Classes/Models/AbstractPost.m

```diff
@@ -204,21 +204,16 @@ - (Media *)featuredImage
 - (void)cloneFrom:(AbstractPost *)source
 {
     for (NSString *key in [[[source entity] attributesByName] allKeys]) {
-        if ([key isEqualToString:@"permalink"]) {
-            DDLogInfo(@"Skipping %@", key);
+        if ([key isEqualToString:@"permalink"]) {            
```

Comment:
Since this removes the only action in the `if`, I'd turn it into an `if (!...)` instead of having an empty block and an `else`.